pkg/action/build-uki.go

```diff
@@ -650,6 +669,32 @@ func (b *BuildUKIAction) removeUkiFiles() error {
 	return nil
 }
 
+func (b *BuildUKIAction) getEfiStub() (string, error) {
+	if utils.IsAmd64(b.arch) {
```

Comment:
why not a function from BuildUKIAction?

but this is fine, just interested to knwo

Reply:
Is very generic no? Make sense to have it in common as it's not tied to any struct and works everywhere for all cases